rct229/ruletest_engine/ruletest_jsons/scripts/excel_to_test_json.py

```diff
@@ -30,7 +30,7 @@
 update_unit_convention_record(spreadsheet_name, sheet_name)
 
 # Check generated master JSON against latest schema
-json_dir = os.path.join(os.path.dirname(__file__), "..", rule_set)
+json_dir = os.path.join(os.path.dirname(__file__), "../../../../../../..", rule_set)
```

Comment:
Could you check if this change was intended?

Reply:
thanks, did not intend to commit this change